packages/commercetools/api-client/src/extensions/emailExtension.ts

```diff
@@ -0,0 +1,23 @@
+import { ApiClientExtension } from '@vue-storefront/core';
+import { customerCreatePasswordResetToken } from '../api';
+
+export const emailExtension: ApiClientExtension = {
+  name: 'logMailExtension',
+
+  extendApiMethods: {
+    async customerCreatePasswordResetToken(context, email) {
+      const response = await customerCreatePasswordResetToken(context, email);
+      const token = response?.data?.customerCreatePasswordResetToken?.value;
+
+      const emailObject = {
+        to: email,
+        from: 'password-recovery@vue-storefront.io',
+        subject: `Password recovery for ${email}`,
+        html: `<a href='/reset-password?token=${token}'>Reset your password by clicking this link</a>`
+      };
+
+      console.log(JSON.stringify(emailObject));
```

Comment:
Should console.log be here?

Reply:
I moved this code from another place and was thinking about removing it, but after speaking with @Baroshem who wrote this, we agree to leave it for now. I don't think it should be here and there is a task to replace it with something better on the theme level (task EN-447).